torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2292,6 +2292,17 @@ def sample_inputs_leaky_relu(op_info, device, dtype, requires_grad):
                requires_grad=requires_grad)) for _ in range(1, N)]
     return tensors
 
+def sample_inputs_avgpool2d(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+
+    cases = (((1, 3, 9, 9), (3, 3), (1, 1), (1, ), True, False, 2),
+             ((1, 1, 4, 4), (2, 2), (1, 1), (0, ), False, True, -2),
+             ((1, 2, 6, 6), (4, 4), (2, 2), (2, ), True, True, 3))
```

Comment:
Let's add some more cases.
- Could we have one that is just a single kernel size `((1, 3, 9, 9),)`?
- avgpool_2d advertises that stride, padding, and kernel_size can be a single number, so we should test something like `((1, 3, 9, 9), 3, 1, 1)`
- Could we have a case where the two strides are not the same and another where the two paddings are not the same?
- We aren't testing division_override=None

Reply:
Thanks for taking a look @zou3519! I've addressed the suggestions in the latest commit. :)